KokkieH/random-quote-spa
# Create app UI

Completes the markup task in #1 

This PR does the following:

* Adds media files in the app `/src` folder
* Creates the HTML markup, including an interactive image that will serve as the clickable element to display a new quote (for now clicking it merely displays a notice in the browser)
* Adds CSS styling.

## Testing instructions

- Check out this branch on your local environment
- In the folder, `/random-quote-spa/random-quote-generator` run `npm install`
- run `npm start`
- View the app at http://localhost:3000/
- The app should appear like this (for now it will only display a static quote):

![Random_Quote_Generator_and_App_js_—_random-quote-spa](https://user-images.githubusercontent.com/11873759/159269969-f81f9f70-17d3-4aa6-af16-fb6fd4cc34f4.png)

- The "inkpot" button should dip the quill on hover:

![Random_Quote_Generator](https://user-images.githubusercontent.com/11873759/159272014-b4cf8ed6-5710-4bcf-885c-3982a11a8e95.png)

- Also make sure the button is clickable. When clicked, it should display a notice in the browser like this:
![Random_Quote_Generator](https://user-images.githubusercontent.com/11873759/159270470-d85f0662-f7ad-4ad5-a5be-50c5a82d5e7b.png)
